### DISCUSSION TITLE
GetSceneList gives a list of scenes in reverse order, (sceneIndex inversed)

### DISCUSSION THREAD

**Discussion by henry-delaruelle** [Palakis/obs-websocket#1022] [Q&A]
After a GetSceneList request i get a list of al the scenes, but it seems that the in sceneIndex is reversed, the last scene in OBS has a sceneIndex 0. the first scene has the highest index.
Is it supposed to work this way or will it change in a newer version ?

**Comment by tt2468** [comment]
Index 0 starts at the bottom of the list to match the behavior of scene items.
